dev/dags/asset_triggered_dags.yml

```diff
@@ -0,0 +1,25 @@
+default:
```

Comment:
Were we able to test with Airflow standalone or Astro deployment that this dag works well since the CI is disabling to run this DAG?

Reply:
Yes, worked fine with astro-cli.

in our repo, `make docker-run`